docs/node-driver.md

```diff
@@ -0,0 +1,49 @@
+#  Harvester node driver
+
+Rancher can dynamically provision nodes in Harvester and install Kubernetes on them by the Harvester node driver.
+
+> Available as of v0.2.0
+
+## Add Harvester node driver
+
+### ISO mode
+
+In ISO mode installation, Harvester node driver is built in, so you don't have to add it manually.
+
+### App mode 
+
+1. From the **Global** view, choose **Tools > Drivers** in the navigation bar. From the **Drivers** page, select the **Node Drivers** tab. In version before v2.2.0, you can select **Node Drivers** directly in the navigation bar.
+2. Click **Add Node Driver**.
+3. Enter **Download URL** and **Custom UI URL**. 
+4. Add domains to the **Whitelist Domains**.
+5. Click **Create**.
+
+You can get all node driver resources from [docker-machine-driver-harvester](https://github.com/harvester/docker-machine-driver-harvester/releases) and [ui-driver-harvester](https://github.com/harvester/ui-driver-harvester/releases) .
+
+## Create Node template
+You can use the Harvester node driver to create node templates and eventually node pools for your Kubernetes cluster.
+
+1. Configure  **Account Access**, for Harvester embedding rancher, you can choose **Internal Harvester**,  which will use the  `harvester.harvester-system` as the default `Host`, `8443` as the default `Port`.
+2. Configure **Instance Options**
+   
+   1). Configure base information, such as `CPUs`, `Memory`, `Disk`, `Disk Bus`.
+   
+   2). Select a `Image` that support `cloud-init`. See [upload image](https://github.com/rancher/harvester/blob/master/docs/upload-image.md) for more info.
+   
+   3). Select a `Network` that rancher server is able to access to the  virtual machines that will be created in Harvester. See [harvester network](https://github.com/rancher/harvester/blob/master/docs/harvester-network.md) for more info.
+   
+   4). Enter the `SSH User`, the username will be used to ssh to nodes. For example, ubuntu cloud image's default user is `ubuntu`.
+   
+3. Enter a **RANCHER TEMPLATE** name.
+
+See [nodes hosted by an infrastructure provider](https://rancher.com/docs/rancher/v2.5/en/cluster-provisioning/rke-clusters/node-pools/) for more info.
+
+## Create cluster
+
```

Comment:
Now users can access the Rancher UI from Harvester, spin up Kubernetes clusters on top of the Harvester cluster, and manage them there. 
> Prerequisite: VLAN network is required for Harvester node driver